### ISSUE TITLE
RCA - Russell St 2

### ISSUE THREAD

**Issue by ak6492** [floodnet-nyc/flood-sensor#61]
**Sensor Deployed:**
- March 5, 2021 by FloodNet

**Sensor Version:** 
- v2.0.

**Firmware Version:**

**Sensor Down:**
- April 3, 2023 19:23

**Solution Implemented:** 
- whole sensor replaced on June 7, 2023 with another v2.0.

**In Lab RCA:** 
- antenna was perfectly intact 
- no visible rusting or corrosion both internally and externally
- battery in sensor completely drained and dead 
- solar panel outputting voltage and increases it's output voltage when light is shined on it thus proving it is functioning correctly 
- sensor booted back up when new battery was inserted into it  

**deployment_id:** easily_dear_mouse

**dev_id:** fs-00023 

**Fixes/Action:**
In conclusion, sensor appears to have gone down due to the battery dying completely and being unable to retain charge from solar panel. The reason behind this seems to be the battery just being out in the field for about two years and going through its natural battery life cycle. Need to track about how long after being out in the field, batteries stop being functional to try to get ahead of sensors going down due to faulty old batteries.

**Comment by mprafull1** [comment]
In Lab RCA:
Battery didn't got charged even when connected to the charger.

**Comment by ak6492** [comment]
Sensor replaced on 06/07, went down again on 06/09, data analysis shows the following: 

![russell 2 rcs](https://github.com/floodnet-nyc/flood-sensor/assets/122378043/feb2041c-cce1-41a4-8fc0-979a004f70d5)

![russell rca 2 x](https://github.com/floodnet-nyc/flood-sensor/assets/122378043/f396af4c-e1d7-4505-8fe2-6bce202e8b47)

![russell rca 2 xx](https://github.com/floodnet-nyc/flood-sensor/assets/122378043/e54f22ab-1f39-4d21-a052-9d25bd5b28f5)

Signal Strength looks good, battery is trending down rapidly and not charging

**Comment by ak6492** [comment]
**07/11 On Site Analysis:** 

battery was visibly puffed up. ended up replacing battery and solar panel.

**Comment by mprafull1** [comment]
Battery was dead also but it charged in the lab.

**Comment by ak6492** [comment]
Sensor data has repeatedly been noisy. On site visit on 11/09 did not reveal any visible sign of what might be causing the noise. Nothing was obstructing the sensor and there was no grass or spiderwebs inside the pipe. Need to observe and understand this issue further. 

![IMG_4752](https://github.com/floodnet-nyc/flood-sensor/assets/122378043/c00aaed5-9fcc-4e34-8cde-09b14a3dfa57)

**Comment by ak6492** [comment]
sensor removed on 02/15
